src/test/java/com/openlattice/rehearsal/edm/EdmTest.kt

```diff
@@ -178,4 +178,15 @@ class EdmTest : MultipleAuthenticatedUsersBase() {
                 edmApi.getEntitySetIds(setOf(es1.name, es2.name, es3.name))
         )
     }
+
+    @Test
+    fun testEdmChangesToEntityTypeTables() {
+        val et = createEntityType()
+        val pt = createPropertyType()
+
+        edmApi.addPropertyTypeToEntityType(et.id, pt.id)
+        edmApi.forceRemovePropertyTypeFromEntityType(et.id, pt.id)
+
+        edmApi.deleteEntityType(et.id)
```

Comment:
does this need to assert anything?

Reply:
I created this test for sanity check only (that it won't error out), when we started to work on entity type tables (prior to ouroboros)